### PR TITLE
Logging and Comments

### PR DESCRIPTION
**Pull request by IanDuncanT**
First part of #25 and #26

### CODE REVIEW

**Comment by IanDuncanT** [comment]
Closes #25 and #26. #26 can be continued on since my comments are basic and not in-depth.

File: commands/asl-node.js [hunk 1]
```diff
@@ -1,6 +1,11 @@
 const { SlashCommandBuilder, EmbedBuilder, parseResponse } = require('discord.js');
 const { table } = require('table');
-const http = require('http');
+const signale = require('signale');
+
+signale.config({displayTimestamp: true, displayDate: true});
+
+// /allstar
+// Replies with license exam stats for the member mentioned or the user who ran the command
```
Comment (nayabw) [inline commands/asl-node.js]: I think this is in the wrong file

**Review by nayabw — APPROVED** [review]

File: commands/asl-node.js (hunk 1 above)
Comment (IanDuncanT) [reply commands/asl-node.js]: Fixed

**Comment by IanDuncanT** [comment]
[Improved Logging](https://trello.com/c/iM5zXg6n/3-improved-logging)

**Pull request merged**